scripts/data_wrangle.Rmd

```diff
@@ -11,7 +11,6 @@ A shiny app of these same data can be found [here](https://hecate.hakai.org/shin
 
 
 ```{r setup}
-library(hakaisalmon) #v0.2.1
```

Comment:
We need to keep this package loaded because it contains the log_cumul_abund function